openreview/profile/management.py

```diff
@@ -30,75 +22,256 @@ def setup(self):
+                                        'type': 'string[]',
+                                        'regex': '[^;,\\n]+(,[^,\\n]+)*'
+                                    }
+                                }
+                            },
+                            'authorids': {
+                                'order': 2,
+                                'value': {
+                                    'param': {
+                                        'type': 'string[]',
+                                        'optional': True
+                                    }
+                                }
+                            },                        
+                            'venue': {
+                                'order': 3,
+                                'description': 'Enter the venue where the paper was published.',
+                                'value': {
+                                    'param': {
+                                        'type': 'string'
+                                    }
+                                }
+                            },
+                            'venueid': {
+                                'order': 4,
+                                'value': {
+                                    'param': {
+                                        'type': "string",
+                                        'const': dblp_group_id,
+                                    }
+                                }
+                            }
+                        }
+                    }                                        
                 }
-            }
-        ))                           
+            )
+        )
+
+        author_coreference_invitation_id = f'{dblp_group_id}/-/Author_Coreference'
+
+        with open(os.path.join(os.path.dirname(__file__), 'process/dblp_author_coreference_pre_process.js'), 'r') as f:
+            file_content = f.read()
+
+        self.client.post_invitation_edit(
+            invitations = meta_invitation_id,
+            signatures = [dblp_group_id],
+            invitation = openreview.api.Invitation(
+                id=author_coreference_invitation_id,
+                readers=['everyone'],
+                writers=[dblp_group_id],
+                signatures=[dblp_group_id],
+                invitees=['~'],
+                preprocess=file_content,
+                edit={
+                    'readers': ['everyone'],
+                    'signatures': { 
+                        'param': { 
+                            'items': [
+                                { 'prefix': '~.*', 'optional': True },
+                                { 'value': self.support_group_id, 'optional': True },
+                                { 'value': dblp_uploader_group_id, 'optional': True } 
+                            ]
+                        } 
+                    },
+                    'writers':  [dblp_group_id, '${2/signatures}'],
```

Comment:
i think we discussed about editing author coreference edits and user may encounter an error when author id is changed because it does not match invitation value

so i think edit.writers need to be updated to disallow editing of author coreference edits

Reply:
good catch! let me fix it

Reply:
fixed!